gradlew.bat

```diff
@@ -0,0 +1,90 @@
+@if "%DEBUG%" == "" @echo off

```

Comment:
Is Remy still on Windows? Do we need to hang on to the bat files?

Reply:
I think we should hang onto them for compatibility reasons (what about Dr. Wollowski or other teams down the line?).

Reply:
I vote require them to install linux (joking)